airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -540,7 +540,7 @@ class IssueMilestones(SemiIncrementalMixin, GithubStream):
     is_sorted = "desc"
     stream_base_params = {
         "state": "all",
-        "sort": "updated",
+        "sort": "updated_at",
```

Comment:
It's incorrect!
it's not field, it's sorting param from here:
https://docs.github.com/en/rest/issues/milestones#list-milestones

In really I don't see such sorting param value `updated` in the documentation, I can only see `due_on`, `completeness`, but `updated` also correctly works - stream is sorted by `updated_at` field

Reply:
Thanks for taking a look into this @grubberr 

As you noticed, the documentation only mentions `due_on` and `completeness`. In my tests, sorting by `updated_at` seems to be the default behavior when there's a `direction` param, but the `sort` param is none of the two options.

Although, there's an exception: `created_at` also works to sort by the `created_at` field. In this case, `created` still defaults to `updated_at`.

Because of that, I think using `updated_at` is more adequate.

[Here's a spreadsheet](https://docs.google.com/spreadsheets/d/1cjl_DZYAJVwTupbV6xCb3GpecEezf6R7Shijw0S23NI/edit?usp=sharing) with the test results using different `sort` params.

You can reproduce all of them with:
```bash
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer TOKEN" \
  "https://api.github.com/repos/airbytehq/airbyte/milestones?state=all&sort=TEST&direction=asc&per_page=100" | jq -r '.[] | [.id, .due_on // "YYYY-MM-DDTHH:MM:SSZ", .created_at, .updated_at, .closed_at] | @tsv'
```